### PR TITLE
gcc: add zstd dependency

### PR DESCRIPTION
**Pull request by vpeter4**
gcc 10 adds zstd compression for its LTO data, that's why zstd library should be build before

### CODE REVIEW

**Comment by jernejsk** [comment]
It would be nice to add `--with-zstd` explicitly. I also think that zstd:host shouldn't depend on toolchain in this case.

**Comment by vpeter4** [comment]
I can add zstd explicitly.
The main issue was with bootstrap stage. Maybe zstd also need to be build to be build separately for bootstrap only?

I'm open for suggestions.

**Comment by jernejsk** [comment]
I talked about this line: https://github.com/LibreELEC/LibreELEC.tv/blob/master/packages/compress/zstd/package.mk#L10
I think it should be changed to ccache:host only, like it's done for xz package.

**Comment by vpeter4** [comment]
Ok, will made those changes and build one image for a test.

**Comment by vpeter4** [comment]
I made few changes and now it should be ok I guess.

File: packages/lang/gcc/package.mk [hunk 1]
```diff
@@ -8,9 +8,9 @@ PKG_SHA256="b6898a23844b656f1b68691c5c012036c2e694ac4b53a8918d4712ad876e7ea2"
 PKG_LICENSE="GPL"
 PKG_SITE="http://gcc.gnu.org/"
 PKG_URL="http://ftpmirror.gnu.org/gcc/$PKG_NAME-$PKG_VERSION/$PKG_NAME-$PKG_VERSION.tar.xz"
-PKG_DEPENDS_BOOTSTRAP="ccache:host autoconf:host binutils:host gmp:host mpfr:host mpc:host"
-PKG_DEPENDS_TARGET="toolchain"
-PKG_DEPENDS_HOST="ccache:host autoconf:host binutils:host gmp:host mpfr:host mpc:host glibc"
+PKG_DEPENDS_BOOTSTRAP="ccache:host autoconf:host binutils:host gmp:host mpfr:host mpc:host zstd:host"
+PKG_DEPENDS_TARGET="toolchain zstd"
```
Comment (HiassofT) [inline packages/lang/gcc/package.mk:12]: Is the target dependency really needed?

target is only used to build libgcc/libstdc++ and as we don't build gcc for the target I don't see why there could be any use of zstd code on the target devices.

Comment (vpeter4) [reply packages/lang/gcc/package.mk:12]: Will remove it then.

**Pull request merged**